src/emulator/renderer/src/texture.cpp

```diff
@@ -38,8 +38,8 @@ void configure_bound_texture(const SceGxmTexture &gxm_texture) {
     glTexParameteriv(GL_TEXTURE_2D, GL_TEXTURE_SWIZZLE_RGBA, swizzle);
 
     const GLenum internal_format = translate_internal_format(fmt);
-    const unsigned int width = gxm::get_width(&gxm_texture);
-    const unsigned int height = gxm::get_height(&gxm_texture);
+    const unsigned int width = static_cast<decltype(width)>(gxm::get_width(&gxm_texture));
```

Comment:
```suggestion
    const auto width = static_cast<uint32_t>(gxm::get_width(&gxm_texture));
```
same in 3 cases below
better avoid `decltype` when we can